ctrd/container.go

```diff
@@ -540,7 +541,7 @@ func (c *Client) createContainer(ctx context.Context, ref, id, checkpointDir str
 		containerd.WithContainerLabels(container.Labels),
 		containerd.WithRuntime(fmt.Sprintf("io.containerd.runtime.v1.%s", runtime.GOOS), &runctypes.RuncOptions{
 			Runtime:       container.Runtime,
-			RuntimeRoot:   runtimeRoot,
+			RuntimeRoot:   path.Join(runtimeRoot, container.Runtime),
```

Comment:
Do we first create this directory or someone would create it if non-exist?
Just confirm.

Reply:
> Do we first create this directory or someone would create it if non-exist?
> Just confirm.

The existence of the path and the security of the join are promised by runtime. 😄